### PR TITLE
Updates made by Running Child

### PR DESCRIPTION
**Pull request by AzeAstro**
I added #Start and #End tags where I edited or added commands.

I added in bot.py:
1)Load <cog name>(Users in admins_list only)
2)Reload <cog name>(Users in admins_list only)
3)Unload <cog name>(Users in admins_list only)

Added in cogs:
1)Do and Make: These two are kinda fun commands. like: `py/make me sandwich` and  it returns random answer.

2)OSU Command combinations(in Misc.py) Returns user info according to given options
usage:
py/osu <option:required> <username:required> <mode:optional>
option can be best/recent/user
mode can be standart/taiko/ctb/mania
**DON'T FORGET TO REPLACE os.environ['OSU_TOKEN'] WITH UR API TOKEN FROM OSU API**

3)Needed things:
1)Avatar: Ig it doesn't need explaination
1)Userinfo: Returns user info in embed
2)Serverinfo:  Returns server info in embed
3)Vote: Makes vote embed for 2 options. usage:
py/vote <countdown:optional int> <arg1,arg2>

4)quote: Returns quote
5)Anilist: Returns detailed info about anime/manga. Usage:
py/<anilist or aniinfo> <anime/manga(one of them)> <name>
Note about why I didn't use MyAnimeList API:
MyAnimeList has API Token which is expiring after 1 year but anilist is modern + uses graphiql + required no token + (and ofc I coudln't work with MyAnimeList's API xD )

4)Memes
1)Meme: Returns meme from r/Memes
2)Animeme: Returns animeme from r/Animemes
3)Cursed: Returns cursed comment pic from r/cursedcursedcomments
r/Cursed required Reddit API and it would make it slow + creepy. so ig r/cursedcursedcomments is best desicion for now.

5)Status:
1)Status: Changes status and saves it(for applying status on restart,add it in commands.Bot part)
usage:
py/status <status>

## Checklist

- [x] If code changes were made then they have been tested.
- [x] This PR's changes are cogs related.
  - [x] This PR adds a command.
  - [ ] This PR removes a command.
  - [x] This PR modifies an existing command's code.
- [ ] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

File: cogs/misc.py [hunk 1]
```diff
@@ -0,0 +1,375 @@
...
+	def __init__(self, client):
+		self.client = client
+
+
+	@commands.command(aliases=["osu","Osu","osuinfo","Osuinfo","OsuInfo"])
+	async def osu_cmd(self, ctx,option,user,mode="standart"):
+		if option.lower()=="user":
+			if mode.lower()=="standart":
+				async with aiohttp.ClientSession() as cs:
+					async with cs.get(f"https://osu.ppy.sh/api/get_user?u={user}&k=8cc26d81bf2e174deea7b8d8a504279c07ba9998") as r:
```
Comment (vierofernando) [inline cogs/misc.py:20]: put the API key as an environment variable instead, e.g `os.environ['OSU_TOKEN']`

**Review by vierofernando — CHANGES_REQUESTED** [review]
good pr, but there are some parts of the code that are better changed before merging

File: cogs/misc.py (hunk 1 above)
Comment (AzeAstro) [reply cogs/misc.py:20]: no matters really.

File: cogs/memes.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    def __init__(self, client):
+        self.client = client
+
+    @commands.command(aliases=["meme","Meme"])
+    async def meme_cmd(self,ctx):
+        async with aiohttp.ClientSession() as cs:
+            async with cs.get('https://www.reddit.com/r/memes.json') as r:
+                res = await r.json()
+                list_posts=range(0,25)
+                x=random.choice(list_posts)
```
[the remark this answers is not in the document]
Comment (AzeAstro) [reply cogs/memes.py:18]: but what will change?

File: cogs/do_make.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+import random
+my_id=339660347499872256
+iljin_id=441929337562988584
+
+answers=["OFC SENPAI!","Sure thing dude...","Yes","Okay","Sounds good","Maybe","I will try","Idk","Maybe no?","No","Definelly no","Nein","I SAID NO U MOTHERFUCKER"]
+class Do_make(commands.Cog):
+    def __init__(self, client):
+        self.client=client
+        
+    @commands.command(aliases=["do","Do"])
```
[the remark this answers is not in the document]
Comment (AzeAstro) [reply cogs/do_make.py:13]: I didn't saw it in commands.Bot :sweat_smile:

File: bot.py [hunk 4]
```diff
@@ -9,4 +9,44 @@
 for cog in filter(lambda c: c.endswith(".py"), listdir("cogs/")):
     bot.load_extension(f"cogs.{cog[:-3]}")
 
+#Added by Running Child
+#Start
+admins_list=[662334026098409480,339660347499872256,286591003794604034]
+async def load_cmd(ctx, extension):
```
[the remark this answers is not in the document]
Comment (AzeAstro) [reply bot.py:15]: right :sweat_smile:

File: cogs/memes.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (vierofernando) [reply cogs/memes.py:18]: replace
```
list_posts=range(0,25)
x=random.choice(list_posts) 
```
with
```
x = random.randint(0, 25)
```

File: cogs/misc.py (hunk 1 above)
Comment (vierofernando) [reply cogs/misc.py:20]: alright.

**Comment by shuanaongithub** [comment]
Thanks.

**Pull request merged**